ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -149,6 +149,7 @@ function ListRules(props) {
   const [dateTimePickerError, setDateTimePickerError] = useState(null);
```

Comment:
And this one as well...it looks like this might be for a different issue?

Reply:
Changes to this file are related to previous PRs ( https://github.com/mozilla-releng/balrog/pull/2999, https://github.com/mozilla-releng/balrog/pull/3002, ) I made, approved but not merged !

@bhearsum